privatelink/aws/terraform/privatelink.tf

```diff
@@ -47,9 +47,13 @@ data "aws_availability_zone" "privatelink" {
   zone_id = each.key
 }
 
+locals {
+  bootstrap_prefix = split(".", var.bootstrap)[0]
+}
+
 resource "aws_security_group" "privatelink" {
-  name = "ccloud-privatelink"
-  description = "Confluent Cloud Private Link minimal security group"
+  name = "ccloud-privatelink_${local.bootstrap_prefix}_${var.vpc_id}"
```

Comment:
A comment above this might be useful for future readers to understand the reasoning.

Reply:
```suggestion
  # Ensure SG uniqueness adding a suffix with cluster id and VPC id. This also helps identifying the resource.
  name = "ccloud-privatelink_${local.bootstrap_prefix}_${var.vpc_id}"
```

Reply:
yep, added a comment